libportability-gfx/src/conv.rs

```diff
@@ -354,3 +366,66 @@ pub fn map_err_device_creation(err: adapter::DeviceCreationError) -> VkResult {
         DeviceLost => VkResult::VK_ERROR_DEVICE_LOST,
     }
 }
+
+pub fn map_attachment_load_op(op: VkAttachmentLoadOp) -> pass::AttachmentLoadOp {
+    match op {
+        VkAttachmentLoadOp::VK_ATTACHMENT_LOAD_OP_LOAD => pass::AttachmentLoadOp::Load,
+        VkAttachmentLoadOp::VK_ATTACHMENT_LOAD_OP_CLEAR => pass::AttachmentLoadOp::Clear,
+        VkAttachmentLoadOp::VK_ATTACHMENT_LOAD_OP_DONT_CARE => pass::AttachmentLoadOp::DontCare,
```

Comment:
I wonder if we should use `Option<LoadOp>` there, with the operation being load/clear
this is minor though...